puppet/modules/quickstack/manifests/neutron/all.pp

```diff
@@ -73,6 +73,11 @@
   $n1kv_vsm_ip                   = undef,
   $n1kv_vsm_password             = undef,
   $n1kv_vsm_username             = undef,
+  $plumgrid_director_vip         = undef,
```

Comment:
I find this point slightly concerning, as we have chosen pacemaker for managing VIPs over keepalived, having this mixed seems less than ideal at best.